test/script/tskdbdiff.py

```diff
@@ -437,7 +437,27 @@ def normalize_db_entry(line, files_table, vs_parts_table, vs_info_table, fs_info
     ig_groups_seen_index = line.find('INSERT INTO "image_gallery_groups_seen"') > -1 or line.find('INSERT INTO image_gallery_groups_seen ') > -1
     
     parens = line[line.find('(') + 1 : line.rfind(')')]
-    fields_list = list(csv.reader([parens.replace(" ", "")], quotechar="'"))[0]
+    no_space_parens = parens.replace(" ", "")
+    fields_list = list(csv.reader([no_space_parens], quotechar="'"))[0]
+    #Add back in the quotechar for values that were originally wrapped (csv reader consumes this character)
+    fields_list_with_quotes = []
+    ptr = 0
+    for field in fields_list:
+        if(len(field) == 0):
+            field = "'" + field + "'"
+        else:
+            start = no_space_parens.find(field, ptr)
+            if(start != -1):
```

Comment:
I believe this condition is unnecessary as start - 1 will never be greater than or equal to 0 if start was equal to -1